src/extensions/unitstable/Units/metricPlugins/common.ts

```diff
@@ -16,5 +21,29 @@ export interface MetricPlugin {
         job_handler_name?: string
     }
     component: React.ComponentType<{record: any}>
+    // comparer: (data: {[key: string]: any}) => (aval: number, bval: number, sortAscending: boolean) => number
+    getRecordValue: (r: any) => MetricValue
     development?: boolean
-}
\ No newline at end of file
+}
+
+const baseNumericComparer = (a: number, b: number, sortAscending: boolean): number => {
+    // if both values are NaN, mark them equal; otherwise sort NaNs after any numeric values.
+    if (isNaN(a) && isNaN(b)) return 0
+    if (isNaN(a)) return -1
+    if (isNaN(b)) return 1
+    // Don't mess with the ordering of two infinities; otherwise sort infinity as greater than finite values
+    if (a === Infinity && b === Infinity) return 0
+    if (a === Infinity) return sortAscending ? 1 : -1
+    if (b === Infinity) return sortAscending ? 1 : -1
```

Comment:
if b is infinite and a is not, a should go before b, so I think this should switch the signs of the 1 and -1 in this line.

(We haven't exactly had a lot of opportunity to test this!)